diff --git a/setup.py b/setup.py
index a376dcc..d44006e 100755
--- a/setup.py
+++ b/setup.py
@@ -28,11 +28,11 @@ setup(name='blueshop',
         package_dir={'blueshop': 'bin'},
         scripts=['bin/blueshop'],
         data_files=[
-            ('/var/local/blueshop/db', ['db/users.sqlite3.dump',
+            ('/var/lib/blueshop/db', ['db/users.sqlite3.dump',
                 'db/shop.sqlite3.dump']),
-            ('/var/local/blueshop/private', ['private/stock.xls',
+            ('/var/lib/blueshop/private', ['private/stock.xls',
                 'private/vip.xls', 'private/eu.xls']),
-            ('etc/blueshop', ['cfg/blueshop.cfg',
+            ('/etc/blueshop', ['cfg/blueshop.cfg',
                 'cfg/mail_pricelist.txt',
                 'cfg/mail_promo.txt']),
             ('share/blueshop/views',
