--- a/cfg/blueshop.cfg	2013-03-20 09:21:36.326443522 +0100
+++ b/cfg/blueshop.cfg	2013-03-20 15:18:47.487442774 +0100
@@ -8,18 +8,18 @@
 [path]
 
 # Where the application is installed.
-base = /usr/local/share/blueshop
+base = /usr/share/blueshop
 
 # Where your modified files are installed.
-overlay = /var/local/blueshop/overlay
+overlay = /var/lib/blueshop/overlay
 
 # Where your personal stuff are.
-private = /var/local/blueshop/private
+private = /var/lib/blueshop/private
 
 [db]
 
-shop = /var/local/blueshop/db/shop.sqlite3
-users = /var/local/blueshop/db/users.sqlite3
+shop = /var/lib/blueshop/db/shop.sqlite3
+users = /var/lib/blueshop/db/users.sqlite3
 
 [users]
 
