--- a/bin/blueshop	2013-03-20 17:54:00.924551459 +0100
+++ b/bin/blueshop	2013-03-20 17:58:15.429196360 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#! /usr/bin/python
 # coding: utf-8
 
 # Copyright (C) 2012 Enrico Rossi
@@ -32,14 +32,17 @@
 
 # Please patch this to fit your installation.
 # It is required that all the modules are found by the application.
-import blueshop
-module_path = os.path.dirname(blueshop.__file__)
+# Fixed by debian installation.
+#import blueshop
+module_path = os.path.dirname(__file__)
+#module_path = "/usr/share/blueshop"
+print module_path
 sys.path.insert(0, module_path)
 
-from blueshop import config, version
-from blueshop import admin_controller, parser_controller, mail_controller
-from blueshop import users_controller, shop_controller
-from blueshop import shop_model, user_model
+import config, version
+import admin_controller, parser_controller, mail_controller
+import users_controller, shop_controller
+import shop_model, user_model
 
 __version__ = version._version_
 
